webapp/javascript/components/TimelineChart/TimelineChartPlugin.ts

```diff
@@ -180,6 +162,12 @@ import { getUTCdate } from '@webapp/util/formatDate';
     };
 
     function bindEvents(plot, eventHolder) {
+      const o = plot.getOptions();
+      // don't initiate logic if there is exploreTooltip in props
```

Comment:
I can see we would return early if `exploreTooltip` is set, so the comment is a bit redundant. The real question here is why do we need to return early. Why does the logic below should NOT be initialized?